### ISSUE TITLE
Misspelled word "successfully"

### ISSUE THREAD

**Issue by GabrieleKaceviciute** [flexn-io/renative#1574]
**Describe the bug**
Misspelled word "successfully" on iOS, tvOS, webOS target launch summary (there may be more places).

**To Reproduce**
1. Go to template-starter or app-harness packages
2. do `npx rnv target launch -p ios -t <simulator-name>` -> observe ( `info: ✔ Succesfully launched iPhone 15`)

**Expected behavior**
Word "Succesfully" should be replaced "Successfully"

**Screenshots or copy&paste**
```
➜  template-starter git:(1.0.0-rc.19) ✗ npx rnv target launch -p ios -t iPhone\ 15

┌─────────────────────────────────────────────────────────────────┐
│ ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗ │
│ ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝ │
│ ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗   │
│ ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝   │
│ ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗ │
│ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝ │
│ 🚀 v:1.0.0-rc.19 | renative.org | 5/23/2024, 5:07:50 PM         │
│ $ rnv target launch -p ios -t iPhone 15                         │
└─────────────────────────────────────────────────────────────────┘
info: Current engine: @rnv/engine-rn (../engine-rn)
task: ○ target launch ↺1
task: ○ workspace configure ↺1
task: ✔ workspace configure
✔ Executing: xcrun simctl list --json
info: Launching: iPhone 15 (use -t to use different target)...
info: ✔ Succesfully launched iPhone 15
task: ✔ target launch
┌─────────────────────────────────────────────────────────────────┐
│ ✔ SUMMARY | 5/23/2024, 5:07:50 PM | rnv@1.0.0-rc.19             │
│ $ rnv target launch -p ios -t iPhone 15                         │
├─────────────────────────────────────────────────────────────────┤
│ Project: @rnv/template-starter@1.0.0-rc.19                      │
│ Workspace: rnv                                                  │
│ Platform (-p): ios                                              │
│ Engine: @rnv/engine-rn                                          │
│ App Config (-c): template                                       │
│ Build Scheme (-s): debug                                        │
│ Env Info: darwin | arm64 | node v18.19.0                        │
│ Executed Time: 0h:0m:0s:672ms                                   │
│ info: ✔ Succesfully launched iPhone 15
└─────────────────────────────────────────────────────────────────┘
```

**Desktop (please complete the following information):**
- OS: macOS Ventura 13.6.4 (M2)
- Node Version 18.19.0
- RNV Version 1.0.0-rc.19

**Comment by GabrieleKaceviciute** [comment]
Note: "Template" is misspelled on new project creating -> 

![tempate](https://github.com/flexn-io/renative/assets/106537292/ca33fa0f-361f-4352-869a-046ebfd77ad6)

**Comment by GabrieleKaceviciute** [comment]
fixed on 1.0.0-rc.21
